### PR TITLE
feat: add mobile event plugin

### PR DESCRIPTION
**Pull request by tangying1027**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / Document optimization
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/g-mobile/src/dom.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
+import { isFunction, isNumber } from '@antv/util';
```
Comment (zengyue) [inline packages/g-mobile/src/dom.ts:1]: 这个文件再看下？下面很多方法应该是不需要的

File: packages/g-plugin-mobile-interaction/src/MobileInteractionPlugin.ts [hunk 2]
```diff
@@ -35,75 +36,30 @@ export class MobileInteractionPlugin implements RenderingPlugin {
...
-      // // @see https://zhuanlan.zhihu.com/p/24555031
-      // $el.addEventListener('wheel', onPointerWheel, {
-      //   passive: true,
-      //   capture: true,
-      // });
+      canvasEl.addEventListener('touchstart', onPointerDown, true);
+      canvasEl.addEventListener('touchend', onPointerUp, true);
+      canvasEl.addEventListener('touchmove', onPointerMove, true);
+
+      self.document.addEventListener('mousemove', onPointerMove, true);
```
Comment (zengyue) [inline packages/g-plugin-mobile-interaction/src/MobileInteractionPlugin.ts:45]: self 这里会指向 window 对象，小程序上是没有的，删掉吧，下面的也一样

File: packages/g-plugin-mobile-interaction/package.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "name": "@antv/g-plugin-mobile-interaction",
   "version": "0.0.1",
   "description": "A G plugin listening events in mobile environment",
-  "main": "lib/index.js",
-  "module": "es/index.js",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
```
Comment (zengyue) [inline packages/g-plugin-mobile-interaction/package.json:6]: 这个不要提交

File: packages/g-mobile/src/dom.ts (hunk 1 above)
Comment (tangying1027) [reply packages/g-mobile/src/dom.ts]: 已修改

File: packages/g-plugin-mobile-interaction/src/MobileInteractionPlugin.ts (hunk 2 above)
Comment (tangying1027) [reply packages/g-plugin-mobile-interaction/src/MobileInteractionPlugin.ts]: 已修改

File: packages/g-plugin-mobile-interaction/package.json (hunk 3 above)
Comment (tangying1027) [reply packages/g-plugin-mobile-interaction/package.json]: 已修改

**Pull request merged**